docs/developer/app-store/apps/cms.mdx

```diff
@@ -0,0 +1,183 @@
+:::
+
+## Glossary
+
+The app uses the following terms:
+
+### Provider
+
+The provider is a CMS platform that is supported by the app.
+
+### Provider Configuration
+
+Provider configuration is an instance of the configured provider.
+Usually, it contains credentials and field mapping. The app allows multiple configurations of the same provider.
+
+For example, you can connect your CMS platform's production and staging setup or have different configurations for each channel.
+
+### Provider-channel connection
+
+The app allows you to connect a provider configuration to the Saleor channel.
+Once it happens, product operations in Saleor will automatically send product updates to the provider with chosen configuration.
+
+This step is optional, but you will likely want to connect the provider configuration to the channel.
+
+:::caution Connecting many channels
+If you want to connect many channels to the same provider configuration, you must create a separate connection for each channel using the same provider configuration.
+:::
+
+### Initial sync/bulk sync
+
+Once the app is installed and the connection is configured, the App will only be aware of product operations that happen after it was installed.
+
+To synchronize all the products that were created before the app was installed, you must perform an initial sync.
+
+Once the connection is configured, the app will allow you to use this connection and scan the entire Saleor database for products and send them to the provider.
+
+:::caution Limitation
+At the moment, the App is performing bulk sync on the frontend. That means when this is happening, the App must stay open. Closing the window will stop the operation.
+:::
+
+## Configuration
+
+To configure the app, follow the instructions built into the app.
+
+Usually, you will need to perform the following steps:
+
+1. Create a provider configuration for the chosen provider.
+2. Create a connection of this provider to the selected channel.
+3. Perform an initial sync.
+4. Edit or create some product variant to see if it is synchronized with the provider.
+
+### CMS fields setup
+
+The App allows you to map Saleor product fields to the CMS fields.
+Some providers allow the App to autocomplete available fields. If autocomplete is not available, you must ensure fields are entered without typos.
+
+## Providers
+
+### Contentful
+
+- Fields are autocompleted; you must provide [space ID](https://www.contentful.com/help/find-space-id/) and a [token](https://www.contentful.com/help/personal-access-tokens/).
+- Entries will be created with an ID that matches Saleor product variant ID.
+- Bulk sync operations are limited to 2 items per second due to Contentful API limitations.
+- All fields must be the type of `string/text`, except `channels`, which should be type `json`.
+
+### DatoCMS
+
+- Fields are autocompleted. You must provide a full-access API token (Content Management API).
```

Comment:
**polish**: same as above regarding splitting these two sentences into separate points.

Also, we should be linking to DatoCMS documentation about the API token.